azure-pipelines.yml

```diff
@@ -40,7 +40,8 @@ jobs:
     displayName: 'Maven Build'
     inputs:
       goals: 'install'
-      options: '-B --settings azure-mvn-settings.xml -Dnative-image.docker-build -Dtest-postgresql -Dnative-image.xmx=6g -Dnative -Dno-format'
+      mavenAuthenticateFeed: true
+      options: '-B  -Dnative-image.docker-build -Dtest-postgresql -Dnative-image.xmx=6g -Dnative -Dno-format'
```

Comment:
Is it normal that you remove the settings? It's automatically taken care of?

Reply:
I was testing a setting that should have taken care of it, but it looks like this is only for deployments. At the moment it does not look like there is any way to do this without leaking auth credentials to the maven feed. If we can create a read only token this may not be an issue but it needs more investigation.

Basically if we use the azure 'secrets' thing to store the auth credentials any user can submit a PR that changes the azure yaml to just output the secret, so it is not secure.